virttest/utils_test/libvirt.py

```diff
@@ -3620,11 +3621,22 @@ def create_secret(params):
     logging.debug("The secret xml is: %s" % sec_xml)
 
     # define the secret and get its uuid
-    ret = virsh.secret_define(sec_xml.xml)
+    if remote_args:
+        server_ip = remote_args.get("remote_ip", "")
+        server_user = remote_args.get("remote_user", "")
+        server_pwd = remote_args.get("remote_pwd", "")
+        remote_virsh_session = virsh.VirshPersistent(**remote_args)
```

Comment:
Better do some basic checks for parameter whether they are "" before .

Reply:
Done.